packages/clayui.com/src/pages/docs/getting_started/cdn-links.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "CDN Links"
+description: ""
+layout: "guide"
+weight: 3
+---
+
+<article id="cdn-links">
+
+Feel free to choose a specific version of the [CDN](https://www.jsdelivr.com/package/npm/clay) if you want to keep track of the changes we make. But if you feel at ease and want to keep Clay always up to date, move on.
+
+## CSS
+
+> Use any of the available automatic cdn distributions and copy-paste the stylesheet `<link>` into your `<head>` before all other stylesheets to load our Atlas skin.
+
+```text/html
+https://cdn.jsdelivr.net/npm/clay/lib/css/atlas.css
```

Comment:
hey @jbalsas, Unfortunately I do not get to solve the problem of Electric escaping the HTML, the code is passed to the [ElectricCode](https://github.com/electricjs/electric/blob/master/packages/electric-clay-components/src/ElectricCode.soy#L20) component but as it is rendered at compile time it prints like any tag and when loaded by the browser it understands it as a HTML tag and not just string, when we move to an invalid URL this works because it does not load the js file and does not break the page, but the same behavior happens, it tries to download the file and then the codemirror resolves in runtime.

Reply:
Yeah, I figured... it's fine for now, let's see how this is handled when we move to a different site generator 🤷‍♂️

Reply:
One way to avoid this was to format the code at compile time instead of at runtime... That's what Gatsby does along with the prismjs plugin.